src/Material/GammaLawGas.cc

```diff
@@ -53,7 +53,7 @@ setPressure(Field<Dimension, Scalar>& Pressure,
             const Field<Dimension, Scalar>& massDensity,
             const Field<Dimension, Scalar>& specificThermalEnergy) const {
   CHECK(valid());
-  for (size_t i = 0; i != massDensity.numElements(); ++i) {
+  for (auto i = 0; i != massDensity.numElements(); ++i) {
```

Comment:
`auto` evaluates to a signed integer here which throws a sign compatibility warning between `i` and `massDensity.numElements()` since `i` will never be negative we should keep it as an unsigned type. This is also seen at lines 71, 86 and 159 of this file